src/Main/TextUserInterface.cpp

```diff
@@ -1644,7 +1666,7 @@ namespace VeraCrypt
 
 			// Re-encrypt volume header
 			SecureBuffer newHeaderBuffer (decryptedLayout->GetHeaderSize());
-			Core->ReEncryptVolumeHeaderWithNewSalt (newHeaderBuffer, decryptedLayout->GetHeader(), options.Password, options.Pim, options.Keyfiles);
+			Core->ReEncryptVolumeHeaderWithNewSalt (newHeaderBuffer, decryptedLayout->GetHeader(), options.Password, options.Pim, options.Keyfiles, true);
```

Comment:
Same as above: EMVOption parameter is always set to true regardless of the actual EMV option value.

Reply:
Same as above : VC in CLI mode always checks for EMV cards